GitHub-trainings/github-pr-training/latex-slides/GitHub-PR-training.tex

```diff
@@ -0,0 +1,806 @@
+		\end{itemize}
+		
+	\end{columns}
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Stage: The merge stage}
+	
+	\huge\centering \textbf{MERGE STAGE}
+	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Stage: The merge stage}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		
+		\Large \textbf{The merge stage:}
+		\vspace{1em}
+		\normalsize
+		\begin{itemize}
+			\setlength\itemsep{.5em}
+			\item Once the work stage is complete, the merge stage is quick
+			\item Clean up the task and make sure it was properly documented
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S3.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Merge the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{.5em}
+			\item Make sure that the PR is well documented - this is where future team members will read how something was done and why
+			\item Then you can merge your well-reviewed and well-documented code
+			\item \textbf{Test run your code after merging it!}
+			
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S3-1.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Delete the branch}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Always delete branches after they are merged
+			\item If you want to keep working in a branch with the same name, then re-create that branch
+			\item Otherwise you risk working on an outdated version of the repo
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S3-2.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Close issue}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		If you created an issue for this task:
+		\begin{itemize}
+			\setlength\itemsep{.5em}
+			\item Make sure that the PR number is referenced in the issue. 
+			\item For example: \textit{``Issue resolved in PR \#12''}
+			\item Then close the issue
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S3-3.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+
+
+\section{Part 2: \newline GitFlow - How to fit the ``branch-PR-merge'' cycle into your workflow}
+
+
+\input{../../Common-Resources/slides/GitHub-Links.tex}
+
+\input{../../Common-Resources/slides/GitHub-Commit-URL.tex}
```

Comment:
including this info is an interesting practice. Perhaps we can go even beyond and consider beginning to implement the use of  GH releases for course editions in our course repositories at Analytics!

Reply:
Interesting, but not something for this PR :) - Do you want to create an issue for this?

Reply:
will do